src/runtime.ts

```diff
@@ -631,6 +586,143 @@ export async function createDefaultRuntime(
   };
 }
 
+/**
+ * create new runtime instance
+ *
+ * @param      {any}               web3           connected web3 instance
+ * @param      {DfsInterface}      dfs            interface for retrieving file from dfs
+ * @param      {any}               runtimeConfig  configuration values
+ * @return     {Promise<Runtime>}  runtime instance
+ */
+export async function createDefaultRuntime(
+  web3: any, dfs: DfsInterface, orgRuntimeConfig: any, options: Runtime = { },
+): Promise<Runtime> {
+  const runtimeConfig = cloneDeep(orgRuntimeConfig);
+  const logger = options.logger || (new Logger());
+  let contextLessRuntime;
+
+  // check if mnemonic and password are given
+  if (runtimeConfig.mnemonic && runtimeConfig.password) {
+    const tempConfig: any = await Onboarding.generateRuntimeConfig(
+      runtimeConfig.mnemonic,
+      runtimeConfig.password,
+      web3,
+    );
+    // eslint-disable-next-line prefer-destructuring
+    runtimeConfig.account = Object.keys(tempConfig.accountMap)[0];
+    if (!runtimeConfig.accountMap) {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
why is the eslint comment necessary, when runtimeConfig is not a param?

Reply:
Then i would need to create a new local parameter. Would afterwards something like this

```ts
const [ account ] = Object.keys(tempConfig.accountMap);
runtimeConfig.account = account;
```